steps/create-hive-dynamo-table.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euo pipefail
+(
+# Import the logging functions
+source /opt/emr/logging.sh
+
+function log_wrapper_message() {
+    log_adg_message "$1" "create-hive-dynamo-table.sh" "$$" "Running as: $USER"
+}
+
+
+log_wrapper_message "Creating external hive table"
+
+hive -e "CREATE EXTERNAL TABLE IF NOT EXISTS data_pipeline_metadata_hive (Correlation_Id STRING, Run_Id BIGINT, DataProduct STRING, DateProductRun STRING, Status STRING) \
```

Comment:
What database are you creating this table in? this needs to belong to a database